e2e/main.e2e.spec.ts

```diff
@@ -117,4 +117,25 @@ test.describe.serial('Main App Test', () => {
     await expect(troubleshootingPage.logsButton).toBeVisible();
     await expect(troubleshootingPage.factoryResetButton).toBeVisible();
   });
+
+  test('should open preferences modal when clicking preferences button', async() => {
+    const navPage = new NavPage(page);
+
+    navPage.preferencesButton.click();
+    const modal = page.locator('.modal');
+
+    await expect(navPage.preferencesButton).toBeVisible();
+    await expect(modal).toBeVisible();
+  });
+
+  test('should open preferences modal when using keyboard shortcut', async() => {
+    if (!os.platform().startsWith('darwin')) {
```

Comment:
In this case I would do the positive-test first, just to reduce cognitive overhead:
```
if (os.platform().startsWith('darwin')) {
        page.keyboard.press('Meta+,');
…
```

Reply:
Driving by…
Note that `os.platform()` has a specific list of possible values; `darwin` is the only one of those that starts with the substring `darwin`.  So you can just do `os.platform() === 'darwin'` instead.

(We should actually do that for `win32` too, since they gave up and never had a `win64`)